### ISSUE TITLE
Question to developers of Privacy Badger - (UI broken, messy UX...)

### ISSUE THREAD

**Issue by cypherpunk** [EFForg/privacybadgerfirefox#487]
Respected Developers,

first of all,  thank you for creation of Privacy Badger.

I have no doubt that you will resolve the issues many of us are having with interface since the last update : Thursday August 06, 2015

I also understand  this is clearly made by people who dedicate their free time and I surely appreciate their time and good will. However, until such time arrives, could you kindly post the previous version of Privacy Badger, before being affected with the latest looks?

If you would be so kind to upload the same and let us know of such link. Then we can go back to that version, disable updates, until this is resolved. Like others, I lover PB and I hate the state where I can’t intervene with any of the settings, specially where my settings are ultimately restrictive.

Thank you
Mark

**Comment by cooperq** [comment]
https://www.eff.org/files/privacybadger-0.99.xpi
or
https://www.eff.org/files/privacybadger-0.2.6.xpi

I would actually appreciate if you would download 0.99 first and tell me if you stil lhave the same problems. So I can try to debug this issue.

**Comment by cypherpunk** [comment]
Cooperq,

thanks you for your help. Those links are not complete, but once I learned about the exact version,
it was easy to find them here : 

https://eff.org/files/privacy-badger-0.99.xpi
https://eff.org/files/privacy-badger-0.2.6.xpi

No, I’m afraid, 0.99 behaves in exactly the same fashion as the latest PB. While 0.2.6 brings the same superb functionality back to my FF39.  I have disabled updates for 0.2.6, as I follow your announcements here. If I can assist you in any way, I would gladly install 0.99 back, just tell me what you need from me and I will gladly supply the data.

**Comment by cooperq** [comment]
Which exact UI issue are you having?

**Comment by cypherpunk** [comment]
Cooperq,

precisely this : http://picload.org/image/ipcpwdc/privacybadger06.08.2015-02.png

as described here: https://github.com/EFForg/privacybadgerfirefox/issues/457

exactly the same behavior 0.99/1.0, or simply said,  absolutely frozen UI 

I even like default red color and purity of 0.2.6., but forgive this comment as it is from personal preference, and not related to our issue with UI

**Comment by cooperq** [comment]
Interesting, can you tell me if there are any errors in the privacy badger console for you? You can find them by click on the firefox menu and then developer->browser console and filtering messages for ones thatt say 'privacybadger'

**Comment by cypherpunk** [comment]
Cooperq,

PB 1.0.0 : http://freetexthost.com/jxhrdlh10o 
or even more sorted :  http://freetexthost.com/nq1zo4refl

But guess what, now that tried again 0.99, and I have actually tried more sites, suddenly 0.99 works:
http://picload.org/image/iwgoarp/privacybadger0.99.0-12.08.2015.png

Errors are no strangers in 0.99 : http://freetexthost.com/upeoi2fmjc

but, I repeat, 0.99 is actually working great no issues so far. And looking at the changes, I see the reasons, and I see the proof and they are surely welcomed

**Comment by cooperq** [comment]
interesting, I am going to take a look at the channel.originalURI error today. I also wonder if you have localstorage turned off in your browser (apparently this is a side effect of disabling all cookie access as well). I am working on both of those issues.

**Comment by cypherpunk** [comment]
Well,  on that note:  http://picload.org/image/iwrdgaw/privacybadger12.08.2015-04.png

Looking at my laptop and desktops while using new UI, I have to say, elaborate and much more efficient. 

But, Cooperq, allow me to ask you just one question. I see those small badgers on many forums/sites:  http://picload.org/image/iwrdgai/privacybadger12.08.2015-03.png

and I wonder, since in  0.2.6 this was never seen. Is this due to improvements made in later versions, or could as result of  "glitches" while finalizing of updating present version. Or simply said, should I see those things, or should they be hidden from my view?  Just asking to understand, whether this is normal.

Thanks for your attention

**Comment by cooperq** [comment]
Your issues should be fixed as of 1.0.1 Can you please install the latest version and try again?

**Comment by cypherpunk** [comment]
@Cooperq,

1.0.1 UI starts fine, but PB found no tracker on any site, so that made me suspicious. There was no errors reported, but on every page that was filled with trackers, PB was claiming nothing to declare.
So I went Options/Filter Setting and once I touched a few, then PB came back to life and with proper amount of detection. 

Also, "Filter Settings" takes too much CPU. The moment it is closed CPU goes back to normal. I’ve tried closing FF process to make sure, but it remains CPU hungry whenever I go back to "Filter Settings" also, trying to scroll takes even more CPU power (over 70%) and it has waiting period of several seconds until I’m allowed to move further. Other then that 1.0.1 seems to fully functional
